seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-jdbc/src/main/java/org/apache/seatunnel/flink/jdbc/sink/JdbcSink.java

```diff
@@ -151,4 +178,56 @@ public void outputBatch(FlinkEnvironment env, DataSet<Row> dataSet) {
                 .finish();
         dataSet.output(format);
     }
+
+    @Override
+    public void close() throws Exception {
+        executePostSql();
+    }
+
+    private void executePreSql() {
+        if (!StringUtils.isNotBlank(preSql)) {
+            LOGGER.info("Starting to execute pre sql: \n {}", preSql);
+            try {
+                executeSql(preSql);
+            } catch (SQLException e) {
+                LOGGER.info("Execute pre sql failed", e);
```

Comment:
Should we change the exception log level to warn or error?

Reply:
hi @legendtkl  , thank for your reivew , it's my bad, i will change them soon

Reply:
Use error level is great.